src/main/java/org/folio/circulation/domain/RequestRepresentation.java

```diff
@@ -86,9 +88,10 @@ private static void addAdditionalItemProperties(JsonObject request, Item item) {
       itemSummary.put("status", statusValue);
     }
 
-    String callNumber = item.getCallNumber();
-    if (callNumber != null) {
-      itemSummary.put("callNumber", callNumber);
+    EffectiveCallNumberComponents callNumberComponents = item.getEffectiveCallNumberComponents();
+    if (callNumberComponents != null) {
+      write(itemSummary, CALL_NUMBER, callNumberComponents.getCallNumber());
+      write(itemSummary, EFFECTIVE_CALL_NUMBER_COMPONENTS, JsonObject.mapFrom(callNumberComponents));
```

Comment:
Could this be extracted to a separate `callNumberComponentsRepresentation` class, so that the details of mapping are not duplicated and hidden from the various callers?